Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -8,8 +8,12 @@ import QuartzCore
 class LayoutManager: NSLayoutManager {
 
     /// Blockquote's Left Border Color
-    ///
-    var blockquoteBorderColor: UIColor? = UIColor(red: 0.52, green: 0.65, blue: 0.73, alpha: 1.0)
+    /// Set the array with how many colors you like, they appear in the order they are set in the array.
+    var blockquoteBorderColors = [UIColor(hexString: "9252FF")!,
```

Comment:
As a default I would like to have just a single color like this: `[UIColor.gray]` then users of the library can configure to their needs.

Reply:
oh, I had picked such a nice default color theme though. But yeah, I guess that makes sense

Reply:
should I use `systemGray` instead of `gray` ?

Reply:
At this location because the library still supports iOS11 we should stick to gray that is available on all version of the OS since iOS11.

If you use systemGray then you need to put code to branch it depending of building to iOS 13 or a previous version of the OS.

Reply:
well I thought that but it seems system gray works on older iOS releases, it let me put it without saying had to be iOS 13.

also here it says since iOS 7.0, but documentation says that for all system colors which doesn't make sense https://developer.apple.com/documentation/uikit/uicolor/3173143-systemgray

Reply:
it seems that semantic colors like `systemBackground` require iOS 13 where as the other system colors like `systemRed` or `systemGray` have a fallback / or have been around that allows them to be used since iOS 7, although it depends, `systemIndigo` requires iOS 13 and `systemPurple` iOS 9 etc.
<img width="1968" alt="Screen Shot 2020-01-07 at 4 02 44 PM" src="https://user-images.githubusercontent.com/10858053/71939392-51ae6e80-3167-11ea-828e-f056b60197d0.png">